lib/wibox/widget/textbox.lua

```diff
@@ -107,6 +108,8 @@ end
 -- @treturn number The preferred width.
 -- @treturn number The preferred height.
 function textbox:get_preferred_size_at_dpi(dpi)
+    assert(dpi, "No DPI provided")
```

Comment:
Huh? Why these new asserts? Does this just provide better error messages than what happened before? If so, should this assert perhaps be moved to `setup_dpi`?

Reply:
Ping

Reply:
done